backend/controllers/moviesController.js

```diff
@@ -0,0 +1,44 @@
+
+
+// const services = 'netflix, prime.subscription';
+// const country = 'us';
+// const genres = '878';
+// const genres_relation = 'or';
+
+// const url = `https://streaming-availability.p.rapidapi.com/search/filters?services=netflix,prime.subscription&country=us&genres=10749,35&genres_relation=or`;
+
+
+const options = {
+  method: 'GET',
+  headers: {
+    'X-RapidAPI-Key': 'f267b612b6msh504ab02ca095331p142270jsn257fad83de1e',
+    'X-RapidAPI-Host': 'streaming-availability.p.rapidapi.com'
+  }
+};
+
+const MoviesController = {
+  getByFilters: async (req, res, next) => {
+    // const genres = '10749,35';
+    // const nextCursor = '113478:조선명탐정: 흡혈괴마의 비밀';
+    
+    const { genres, nextCursor } = req.params;
+    const services = 'netflix,prime.subscription,prime.rent,prime.buy,apple.rent,apple.buy,hbo,hulu.addon.hbo,prime.addon.hbomaxus,hulu.subscription,hulu.addon.hbo,apple.addon,peacock.free';
+    // const cursor = `&cursor=${nextCursor}`;
+    let cursor;
+    if (nextCursor) cursor = `cursor=${nextCursor}`;
+
+    const url = `https://streaming-availability.p.rapidapi.com/search/filters?services=${services}&country=us&output_language=en&order_by=original_title&genres=${genres}&genres_relation=or&desc=true&show_type=movie&${cursor}`;
```

Comment:
optional - but for best practices I would move all params outside the url. If we decide to use any of those to filter as a feature. 
When writing code you should always think of ways to make your code easier to adjust.

Reply:
i moved all params into their own constants, but only the filters we're actively setting are destructured/grabbed from req.params